File: Service/MediCare/RMA.Service.MediCare.Contracts/Entities/Medical/SearchInvoiceCriteria.cs
using System;

namespace RMA.Service.MediCare.Contracts.Entities.Medical
{
    public class SearchInvoiceCriteria
    {
        public string PracticeNumber { get; set; }
        public int PractitionerTypeId { get; set; }
        public int InvoiceStatusId { get; set; }
        public string SupplierInvoiceNumber { get; set; }
        public string AccountNumber { get; set; }
        public DateTime? InvoiceDate { get; set; }
        public DateTime? TreatmentFromDate { get; set; }
        public DateTime? TreatmentToDate { get; set; }
        public string ClaimReference { get; set; }
        public int PageSize { get; set; }
        public int PageNumber { get; set; }
        public int SwitchBatchInvoiceStatusId { get; set; }
    }
}

File: Web/RMA-Web-Shell/projects/medicare/src/app/shared/models/search-invoice-criteria.ts
export class SearchInvoiceCriteria {
    practiceNumber: string;
    practitionerTypeId: number;
    invoiceStatusId: number;
    switchBatchInvoiceStatusId: number;
    supplierInvoiceNumber: string;
    accountNumber: string;
    invoiceDate: any;
    treatmentFromDate: any;
    treatmentToDate: any;
    claimReference: string;
    pageNumber: number | null;
    pageSize: number | null;
}